pkg/network/kubemacpool_test.go

```diff
@@ -0,0 +1,184 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+var _ = Describe("Testing kubeMacPool", func() {
+	var _ = Describe("Testing kubeMacPool validation", func() {
```

Comment:
remove the `var _ = ` here no need and please check the describe to `Validation function`